### PR TITLE
Add voice option and rate for TTS

### PR DESCRIPTION
**Pull request by shubhamtibra**
### What do you want to achieve?
- Adding two parameters `voiceRate: 2` and `voiceName: "Rishi"` in appOptions
- There is predefined list of voices available in the browser for each language
- If voiceName is specified then that voice will be used
- If the voiceName is not provided or is incorrect, default voice will be used

### PR Checklist

- [x] I have tested it locally and all functionalities are working fine.
- [x] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

-

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: webplugin/js/app/media/media-service.js [hunk 1]
```diff
@@ -68,6 +68,14 @@ Kommunicate.mediaService = {
             if (textToSpeak) {
                 var utterance = new SpeechSynthesisUtterance(textToSpeak);
                 utterance.lang = appOptions.language || "en-US";
+                utterance.rate = appOptions.voiceRate || 1;
+                if (appOptions.voiceName) {
+                    AVAILABLE_VOICES_FOR_TTS.forEach(voice => {
+                        if(voice.name == appOptions.voiceName) {
+                            utterance.voice = voice;
```
Comment (devashishmamgain) [inline webplugin/js/app/media/media-service.js:75]: Break

**Review by devashishmamgain — CHANGES_REQUESTED** [review]

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 2]
```diff
@@ -658,6 +659,17 @@ var userOverride = {
...
                 document.body.classList.remove("accesibility");
             });
+
+            // the browser call getVoices is async
+            // so we are updating the array whenever they're available
+            if (VOICE_OUTPUT_ENABLED && "speechSynthesis" in window) {
+                AVAILABLE_VOICES_FOR_TTS = speechSynthesis.getVoices();
+                if (speechSynthesis.onvoiceschanged !== undefined) {
+                    speechSynthesis.onvoiceschanged = function () {
+                        AVAILABLE_VOICES_FOR_TTS = speechSynthesis.getVoices();
```
Comment (Prochnost55) [inline webplugin/js/app/mck-sidebox-1.0.js:669]: I am just concerned for following:
1. browser compatibility is limited
Screenshot 2021-02-13 at 10 19 58 AM
2. It is an experimental function.

Is it safe to use ?

File: webplugin/js/app/media/media-service.js (hunk 1 above)
Comment (shubhamtibra) [reply webplugin/js/app/media/media-service.js:92]: Done

File: webplugin/js/app/mck-sidebox-1.0.js (hunk 2 above)
Comment (shubhamtibra) [reply webplugin/js/app/mck-sidebox-1.0.js:834]: Yeah, this will cover the cases when `onvoiceschanged` isn't available.

**Review by devashishmamgain — APPROVED** [review]

**Review by Prochnost55 — APPROVED** [review]

**Pull request merged**